src/omotes_simulator_core/adapter/transforms/esdl_asset_mappers/EsdlAssetAtesMapper.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright (c) 2023. Deltares & TNO
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""Module containing the Esdl to Ates asset mapper class."""
+
+from omotes_simulator_core.entities.assets.ates_cluster import AtesCluster
+from omotes_simulator_core.entities.assets.asset_abstract import AssetAbstract
+from omotes_simulator_core.entities.assets.esdl_asset_object import EsdlAssetObject
+from omotes_simulator_core.simulation.mappers.mappers import EsdlMapperAbstract
+
+
+class EsdlAssetAtesMapper(EsdlMapperAbstract):
+    """Class to map an ESDL asset to a ates entity class."""
+
+    def to_esdl(self, entity: AtesCluster) -> EsdlAssetObject:
```

Comment:
The ATES cluster needs to have all the add_physical_data parts in it from the ATES asset.

Reply:
completed.